docs/sources/configure/metrics-traces-attributes.md

```diff
@@ -279,3 +279,23 @@ Beyla periodically resynchronizes the whole Kubernetes metadata at the frequency
 by this property.
 
 Higher values reduce the load on the Kubernetes API service.
+
+## Extra group attributes
+
+Beyla supports adding custom attributes to metrics by configuring `extra_group_attributes`. You need to add the desired group and an array of attributes to be added to this group.
+
+Currently only `app_kube` group is supported, which characterises attributes specific to the cluster in k8s, e.g. container name, pod name, namespace name, pod uid and others.
+
+For example:
+
+```yaml
+attributes:
+  kubernetes:
+    enable: true
+    resource_labels:
+      app: ["app"]
+  extra_group_attributes:
+    app_kube: ["app"]
```

Comment:
The role of `app_kube` is not clear to me here. Above says that characterises the `k8s_*` attribute set here seems to be referring to a concrete label/annotation suffix

Reply:
Perhaps it would be more flexible to add attributes per specific metric (section) rather than per group, as in attribute selection config? What do you think? I didn't think of this idea when I implemented it

Reply:
IMO the per-group approach is fine. My comment was more focused on the docs, as it is not clear to me how it works.

Reply:
ok, tried to improve the documentation, does the group name `k8s_app_meta` fit better?